Silicon/ElkhartlakePkg/Include/FspUpd.h

```diff
@@ -1,7 +1,30 @@
 /** @file
 
-Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
-SPDX-License-Identifier: BSD-2-Clause-Patent
+Copyright (c) 2021, Intel Corporation. All rights reserved.<BR>

```

Comment:
Is FSP released ?  If so, these files should be from the FSP release package.   And we don't need to keep these FSP header files in the SBL repo.

Reply:
Please refer to your Fsp.inf file.  All files coming from FSP repo should be removed from the SBL repo.

Reply:
Since this is addressed in a different commit, I am fine with that.